photoeditor/src/main/java/ja/burhanrashid52/photoeditor/TextStyleBuilder.java

```diff
@@ -0,0 +1,146 @@
+package ja.burhanrashid52.photoeditor;
+
+import android.graphics.Typeface;
+import android.graphics.drawable.Drawable;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.widget.TextView;
+
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
Need to add java doc for public API which our `TextStyleBuilder` class

Reply:
Ok, but I will add it, but look that it's private on lib, no client could use this by his own